### PR TITLE
make new_from_bank to be only for tests

### PR DESCRIPTION
**Pull request by KirillLykov**
#### Problem

There is a comment that it is useful for tests, instead I added `#[cfg(test)]`.

### CODE REVIEW

File: rpc/src/rpc.rs [hunk 1]
```diff
@@ -348,7 +348,7 @@ impl JsonRpcRequestProcessor {
         )
     }
 
-    // Useful for unit testing
+    #[cfg(test)]
```
Comment (steviez) [inline rpc/src/rpc.rs:358]: nit: We do have some precedent for ending function names like this with `_for_tests`. Ie,
```rust
pub fn new_from_bank_for_tests(
```
That being said, `#[cfg(test)]` makes it functionally true that this is for tests whereas the name is more of a "suggestion" so I'd be ok pushing things as-is

**Review by steviez — COMMENTED** [review]
Good cleanup; lemme know what you think on that one comment.

Looks like `clippy` is failing now since you have a bunch of imports that are now only used in `test` mode. I can give ya another review once you address that

File: rpc/src/rpc.rs [hunk 2]
```diff
@@ -119,6 +114,18 @@ use {
     },
 };
 
+#[cfg(test)]
```
Comment (KirillLykov) [inline rpc/src/rpc.rs:116]: Not sure where to place this, the best place I've found

Comment (steviez) [reply rpc/src/rpc.rs:116]: Seems reasonable

**Review by steviez — APPROVED** [review]

**Pull request merged**